app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -28,7 +23,7 @@ def connect(options = {})
 
   def verify_credentials(auth_type = nil, options = {})
     raise MiqException::MiqHostError, "No credentials defined" if missing_credentials?(auth_type)
-    options[:auth_type] = auth_type
+    options[:auth_type] = 'validate'
```

Comment:
So `'validate'` isn't an auth_type, it is supposed to be `"default"` for your "main" provider then you can have additional credentials for e.g. your metrics endpoint.

Reply:
@agrare I think the validate is used only to check credentials, so I think it isn't the best option to be default

Reply:
No what I mean is `auth_type` is used by `authentication_abcd` to find the authentication credentials you are trying to reference.  If you pass `"validate"` in then it won't be able to find your credentials.

For `verify_credentials` you should connect exactly like you normally would, same `auth_type`.

Reply:
Oooh, got it!!

Reply:
@walteraa, @agrare Firstly thank you for review this PR, I agree with the Adam and I created a new solution.